doc/source/tutorial/stats/hypothesis_pearsonr.md

```diff
@@ -0,0 +1,167 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
++++ {"tags": ["jupyterlite_sphinx_strip"]}
+
+```{eval-rst}
+.. notebooklite:: hypothesis_pearsonr.md
+   :new_tab: True
+```
+
+(hypothesis_pearsonr)=
+
++++
+
+# Pearson's Correlation
+
+Consider the following data from [^1], which studied the relationship between free proline (an amino acid) and total collagen (a protein often found in connective tissue) in unhealthy human livers.
+
+The `x` and `y` arrays below record measurements of the two compounds. The observations are paired: each free proline measurement was taken from the same liver as the total collagen measurement at the same index.
+
+```{code-cell} ipython3
+import numpy as np
+# total collagen (mg/g dry weight of liver)
+x = np.array([7.1, 7.1, 7.2, 8.3, 9.4, 10.5, 11.4])
+# free proline (μ mole/g dry weight of liver)
+y = np.array([2.8, 2.9, 2.8, 2.6, 3.5, 4.6, 5.0])
+```
+
+```{code-cell} ipython3
+import matplotlib.pyplot as plt
+fig, ax = plt.subplots(figsize=(8, 5))
+ax.plot(x, y, '.')
+ax.set_xlabel("total collagen (mg/g)")
+ax.set_ylabel("free proline (μ mole/g)")
+plt.show()
+```
+
+These data were analyzed in [^2] using Spearman’s correlation coefficient, a statistic sensitive to monotonic correlation between the samples. Here, we will analyze the data using Pearson's correlation coefficient, which is sensitive to linear correlation.
```

Comment:
Not mandatory, but just in case, if you want to cite the actual pearsonr function this is how you can do it:

```suggestion
These data were analyzed in [^2] using Spearman’s correlation coefficient, a statistic sensitive to monotonic correlation between the samples. Here, we will analyze the data using Pearson's correlation coefficient (`{class}`scipy.stats.pearsonr`) which is sensitive to linear correlation.
```

As a general rule I've been trying to have at least one link from tutorial to reference, and one from reference to tutorial (those are the See Also links). That makes both the tutorial and the reference docs more visible.